operator/src/main/java/oracle/kubernetes/operator/helpers/ConfigMapHelper.java

```diff
@@ -388,9 +430,29 @@ public NextAction apply(Packet packet) {
             new SitConfigMapContext(
                 this, info.getDomainUid(), getOperatorNamespace(), info.getNamespace(), data);
 
+        // Cannot short circuit here
+        //
+        if (updateDomainResult != null) {
+          LOGGER.fine("ConfigMapHelper:updateDomainResult " + updateDomainResult);
+          if (updateDomainResult.equals("0")) {
+            LOGGER.fine("ConfigMapHelper apply: short circuit finished online update");
+            packet.setDynamicUpdate(true);
```

Comment:
I don't understand this logic.

Reply:
Removed the code, they are for online update